build.gradle

```diff
@@ -67,6 +78,11 @@ task warManualRu(dependsOn: [chopManualRu], type: CreateWar) {
     docLang = 'ru'
 }
 
+task warManualCn(dependsOn: [chopManualEn], type: CreateWar) {
```

Comment:
task `warManualCn` should depend on `chopManualCn`, not En.

Reply:
Hi,thanks.I have fixed that.